PROJECT

```diff
@@ -0,0 +1,7 @@
+version: "2"
+domain: sigs.k8s.io
```

Comment:
`cluster.x-k8s.io`?

Reply:
is `cluster.x-k8s.io` the same used by cluster api?

Reply:
btw, this was just used as a placeholder, but happy to have a sane default from the get go

Reply:
That's the one we're using today in CAPI, and the official one for k8s experimental projects 😄

Reply:
same comment.

Reply:
Cool, will amend